ui/blockly/generators/javascript/clova.js

```diff
@@ -39,3 +39,8 @@ Blockly.JavaScript['clova_get_session_attribute'] = function(block) {
   var code = 'sessionAttribute(' +  text_session_attribute_key + ')';
   return [code, Blockly.JavaScript.ORDER_FUNCTION_CALL];
 };
+
+Blockly.JavaScript['clova_end_session'] = function(block) {
+  var code = 'endSession();\n';
```

Comment:
Don't need to output "\n".

Reply:
Fixed at https://github.com/umakoz/blova/pull/5/commits/a537d8465f17c7bde3abcc2578d77c50c75fbc9b